packages/metadata-common/src/MetadataEditorWidget.tsx

```diff
@@ -86,7 +86,15 @@ export interface IMetadataEditorProps {
    */
   themeManager?: IThemeManager;
 
+  /**
+   * String used to make the title of the editor more readable
+   */
   titleContext?: string;
+
+  /**
+   * Optional field used for the "Save as code snippet" feature
```

Comment:
I would make the description more generic, don't reference the exact use case but a potential one, such as "a default value for the code properties" Also it's worth noting that this would insert this for all code properties if it exists, not sure if that's a feature we want, or if we should try to "aim" where this goes better?

Reply:
Updated the comment - in terms of the behavior, that's worth working on / discussing but this is just getting it back to the state it was in before. As far as our functionality is concerned that wouldn't come up because we'd only be giving the code prop for empty metadata.